vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.java

```diff
@@ -805,6 +807,29 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
             }
         });
 
+
+        final CheckBoxPreference useRageShakeModePref = (CheckBoxPreference) findPreference(PreferencesManager.SETTINGS_USE_RAGE_SHAKE_KEY);
+        final boolean mIsUsedRageShake = PreferencesManager.useRageshake(appContext);
+
+        if(mIsUsedRageShake){
+            useRageShakeModePref.setChecked(true);
+        } else {
+            useBackgroundSyncPref.setChecked(false);
+        }
+
+        useRageShakeModePref.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
+            @Override
+            public boolean onPreferenceChange(Preference preference, Object newValue) {
+                List<MXSession> sessions = Matrix.getMXSessions(getActivity());
+                for (MXSession session : sessions){
+                    session.setUseRageShakeMode((boolean) newValue);
+                }
+
+                return true;
+            }
+        });
+
```

Comment:
There are 2 lines separators whereas the rest of the code uses only one line separator between different blocks of code.